engines/dradis-api/app/controllers/dradis/ce/api/v3/nodes_controller.rb

```diff
@@ -44,7 +44,7 @@ def destroy
       protected
 
       def node_params
-        params.require(:node).permit(:label, :type_id, :parent_id, :position)
+        params.require(:node).permit(:label, :type_id, :parent_id, :position, properties: {})
```

Comment:
This is important, as using JSON opens a can of worms around validation. If you look at the app's NodesController [we stick to raw_properties](https://github.com/dradis/dradis-ce/blob/develop/app/controllers/nodes_controller.rb#L134) to let the user submit a "text blob" with the JSON properties, but we [then apply strict validation](https://github.com/dradis/dradis-ce/blob/develop/app/models/concerns/node_properties.rb#L14) at the model level.

I'd expect for the API flow to follow a very similar pattern.